djangoProject/profiles/views.py

```diff
@@ -1,3 +1,53 @@
-from django.shortcuts import render
+from django.contrib.auth import get_user_model
+from .serializers import StartUpProfileSerializer, InvestorProfileSerializer
+from django.shortcuts import get_object_or_404
+from rest_framework.generics import CreateAPIView, RetrieveAPIView, UpdateAPIView
+from rest_framework.permissions import AllowAny, IsAuthenticated
+from .models import StartUpProfile
 
-# Create your views here.
+User = get_user_model()
+
+
+class InvestorRegistrationView(CreateAPIView):
+    serializer_class = InvestorProfileSerializer
+    permission_classes = [AllowAny]
+
+
+class StartupRegistrationView(CreateAPIView):
+    serializer_class = StartUpProfileSerializer
+    permission_classes = [AllowAny]
+
+    def perform_create(self, serializer):
+        # Extract id_user from query parameters
+        id_user = self.request.query_params.get('id_user')
+        if not id_user:
+            raise ValueError("User ID is missing in the query parameters.")
+
```

Comment:
Throwing ValueError for missing id_user can cause unexpected problems. It is better to use standard DRF errors.

 if not id_user:
            raise ValidationError("User ID is missing in the query parameters.")